extendr-macros/src/pairlist.rs

```diff
@@ -0,0 +1,30 @@
+use proc_macro::TokenStream;
```

Comment:
This is the actual pairlist code.

Reply:
Ah, Ok. For some reason it didn't show up when I did a search for "pairlist".